README.md

```diff
@@ -80,6 +80,40 @@ is.
 End tags must be omitted when possible and attribute values must not be quoted (if necessary, use
 double quotes).
 
+Place one new line between paragraphs (including list elements). Place two or more new lines before
+headings (e.g., `<h3>`). This doesn't apply when a nested heading follows the parent heading.
+
+```html
+<ul>
+ <li><p>Do not place a new line above.
+
+ <li><p>Place a new line above.
+</ul>
+
+<p>Place a new line above.
+
+
+<h3>Place two or more new lines above.</h3>
+
+<h4>Placing one new line is OK here.</h4>
+
+
+<h4>Place two or more new lines above.</h4>
+```
+
+Use camelCase for variable names. On the other hand, it is OK to use spaced names for defined
```

Comment:
I would strike "it is OK to" as it's in fact the style to use spaced names there and it would probably not be okay to use camel case there.

Reply:
Done.